util/types/json/json_test.go

```diff
@@ -64,63 +74,17 @@ func (s *testJSONSuite) TestJSONSerde(c *C) {
 	}
 }
 
-func (s *testJSONSuite) TestParseFromString(c *C) {
-	var jstr1 = `{"a": [1, "2", {"aa": "bb"}, 4, null], "b": true, "c": null}`
-
-	j1, err := ParseFromString(jstr1)
-	c.Assert(err, IsNil)
-
-	var jstr2 = j1.String()
-	c.Assert(jstr2, Equals, `{"a":[1,"2",{"aa":"bb"},4,null],"b":true,"c":null}`)
-}
-
-func (s *testJSONSuite) TestJSONType(c *C) {
-	j1, err := ParseFromString(`{"a": "b"}`)
-	c.Assert(err, IsNil)
-
-	j2, err := ParseFromString(`["a", "b"]`)
-	c.Assert(err, IsNil)
-
-	j3, err := ParseFromString(`3`)
-	c.Assert(err, IsNil)
-
-	j4, err := ParseFromString(`3.0`)
-	c.Assert(err, IsNil)
-
-	j5, err := ParseFromString(`null`)
-	c.Assert(err, IsNil)
-
-	j6, err := ParseFromString(`true`)
-	c.Assert(err, IsNil)
-
-	var jList = []struct {
-		In  JSON
-		Out string
-	}{
-		{j1, "OBJECT"},
-		{j2, "ARRAY"},
-		{j3, "INTEGER"},
-		{j4, "DOUBLE"},
-		{j5, "NULL"},
-		{j6, "BOOLEAN"},
-	}
-
-	for _, j := range jList {
-		c.Assert(j.In.Type(), Equals, j.Out)
-	}
-}
-
 func (s *testJSONSuite) TestCompareJSON(c *C) {
-	jNull, _ := ParseFromString(`null`)
-	jBoolTrue, _ := ParseFromString(`true`)
-	jBoolFalse, _ := ParseFromString(`false`)
-	jIntegerLarge, _ := ParseFromString(`5`)
-	jIntegerSmall, _ := ParseFromString(`3`)
-	jStringLarge, _ := ParseFromString(`"hello, world"`)
-	jStringSmall, _ := ParseFromString(`"hello"`)
-	jArrayLarge, _ := ParseFromString(`["a", "c"]`)
-	jArraySmall, _ := ParseFromString(`["a", "b"]`)
-	jObject, _ := ParseFromString(`{"a": "b"}`)
+	jNull := mustParseFromString(`null`)
+	jBoolTrue := mustParseFromString(`true`)
+	jBoolFalse := mustParseFromString(`false`)
+	jIntegerLarge := mustParseFromString(`5`)
+	jIntegerSmall := mustParseFromString(`3`)
+	jStringLarge := mustParseFromString(`"hello, world"`)
+	jStringSmall := mustParseFromString(`"hello"`)
+	jArrayLarge := mustParseFromString(`["a", "c"]`)
+	jArraySmall := mustParseFromString(`["a", "b"]`)
+	jObject := mustParseFromString(`{"a": "b"}`)
```

Comment:
renamed. but I prefer keep to define strings in here, because they are used many times in testcases.